java/arcs/core/data/ParticleSpec.kt

```diff
@@ -13,6 +13,6 @@ package arcs.core.data
 
 data class ParticleSpec(
```

Comment:
Nit: We could probably use some KDoc on this class.  Pretending to be new to the code base and looking at it, I'm not sure exactly what it's for.

Reply:
Done.